### ISSUE TITLE
Update for 1.0

### ISSUE THREAD

**Issue by Merkury90** [mircearoata/SatisfactoryPakUtilityMod#39]
Any chance your mod will be updated for version 1.0 of the game?
I tried running it as is but could not make it work.

**Comment by mircearoata** [comment]
Yes, I will update the mod, but after SMM is stable. Until then I am fixing issues in SMM

**Comment by Merkury90** [comment]
That's great news. Thanks for your hard work!

**Comment by Dooperlzwak** [comment]
Hi, is this mod ready for 1.0?

**Comment by AdiMajsterek** [comment]
> Hi, is this mod ready for 1.0?

No it isn's still. Just checked in SMM and on ficsit.app

**Comment by FourOne** [comment]
There’s another mod that works for some of the commands we used with this one, but unfortunately, the noclip/ghost mod isn’t working. That’s the only command I really need, as I like to build things in a specific way, and noclip was really helpful for that.

Now that we have a stable SMM, hopefully we’ll get an update soon. Until then, I’ll just have to build the boring way ;)
